internal/core/graphics/image.rs

```diff
@@ -399,6 +430,8 @@ pub enum ImageInner {
     #[cfg(not(target_arch = "wasm32"))]
     BorrowedOpenGLTexture(BorrowedOpenGLTexture) = 6,
     NineSlice(vtable::VRc<OpaqueImageVTable, NineSliceImage>) = 7,
+    #[cfg(feature = "unstable-wgpu-24")]
+    WGPUTexture(vtable::VRc<OpaqueImageVTable, WGPUTexture>) = 8,
```

Comment:
I think it doesn't need the `VRc<OpaqueImageVTable>` Since the `unstable-wgpu-24` will not be enabled for C++.

If necessary, to make sure cbindgen don't generate anything, disable the feature either by adding something is 
https://github.com/slint-ui/slint/blob/e357222e597f9266975029e482aa73938db91d8e/api/cpp/cbindgen.rs#L238
or setting the right flag in the config.parse.expand
(but i think it shouldn't process this feature anyway)

Reply:
Good point. Yeah, I did this for C++, but your suggestion sounds good. I'll give it a try without `VRc`.